tfhe/src/shortint/oprf.rs

```diff
@@ -96,20 +122,54 @@ impl ServerKey {
         let acc: LookupTableOwned =
             self.generate_lookup_table_no_encode(|x| (2 * (x / poly_delta) + 1) * delta / 2);
 
-        let mut ct = self.apply_lookup_table(&ct, &acc);
+        let out_lwe_size = self.bootstrapping_key.output_lwe_dimension().to_lwe_size();
 
-        lwe_ciphertext_plaintext_add_assign(&mut ct.ct, Plaintext((p - 1) * delta / 2));
+        let mut ct = allocate_and_trivially_encrypt_new_lwe_ciphertext(
```

Comment:
same here you can allocate directly a new 0ed ciphertext

Reply:
Done